lib/perl/Genome/Qc/Command/BuildMetrics.pm

```diff
@@ -4,7 +4,8 @@ use strict;
 use warnings;
 
 use Genome;
-use Data::Dumper;
+use YAML::Syck;
+use Test::More qw/no_plan/;
```

Comment:
New commit to remove `Test::More`.  Also tests are now passing.  The failure looked unrelated.